lib/src/client/connection.dart

```diff
@@ -1,9 +1,10 @@
 // Copyright (c) 2018, Juan Mellado. All rights reserved. Use of this source
 // is governed by a MIT-style license that can be found in the LICENSE file.
 
-import 'dart:async' show Future, StreamSubscription;
+import 'dart:async';
```

Comment:
```suggestion
import 'dart:async' show ...;
```
I prefer to import elements, not the full package